src/lib/InputError.svelte

```diff
@@ -1,3 +1,8 @@
-<span class="font-semibold text-red-700">
+<script lang="ts">
+	let className = '';
+	export { className as class };
+</script>
+
+<span class={className}>
```

Comment:
I didn't mean for the classes on there to be removed, instead something like:
```svelte
<span class="font-semibold text-red-700 {className}">
```

Reply:
my bad - got a bit excited there. Will adjust it now!